File: SportskiCentarRKS_API/SportskiCentarRKS_API/ViewModels/RezervacijeResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SportskiCentarRKS_API.ViewModels
{
    public class RezervacijeResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int RezervacijaID { get; internal set; }
            public string nazivProstorije { get; set; }
            public string terminInfo { get; set; }
            public bool Odobrena { get; internal set; }
        }
    }
}

File: Sportski_centar_androidStudio/app/src/main/java/com/example/sports_centar/data/RezervacijeResultVM.java
package com.example.sports_centar.data;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class RezervacijeResultVM implements Serializable {
    public List<Row> rows;

    public class Row implements Serializable
    {
        public int RezervacijaID;
        public String nazivProstorije;
        public String terminInfo;
        public boolean Odobrena;
    }
}
